ckanext/dcatapchharvest/tests/fixtures/1901.xml

```diff
@@ -15,7 +15,12 @@
         <dct:language>fr</dct:language>
         <dct:identifier>346265-fr@bundesamt-fur-statistik-bfs</dct:identifier>
         <dct:format>HTML</dct:format>
-        <dct:rights>NonCommercialAllowed-CommercialWithPermission-ReferenceRequired</dct:rights>
+        <dct:rights>
+          <dct:RightsStatement rdf:about="http://dcat-ap.ch/vocabulary/licenses/terms_by"/>
+        </dct:rights>
+        <dct:license>
+          <dct:LicenseDocument rdf:about="http://dcat-ap.de/def/licenses/cc-zero"/>
+        </dct:license>
         <foaf:page>
```

Comment:
Could we also add tests for the following cases:
- rights field as a string, e.g. `<dcterms:rights rdf:datatype="http://www.w3.org/2001/XMLSchema#string">NonCommercialAllowed-CommercialAllowed-ReferenceRequired</dcterms:rights>`
- license field as a simple URIRef, e.g. `<dcterms:license rdf:resource="http://dcat-ap.ch/vocabulary/licenses/terms_by"/>`

Both of these appear in the Kanton Schaffhausen catalog so it would be useful to test that we handle them right.